src/pages/FeeRateTracker/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react'
+import { useQuery } from 'react-query'
+import { Row } from 'antd'
+import { v2AxiosIns } from '../../service/http/fetcher'
+import styles from './styles.module.scss'
+import Content from '../../components/Content'
+import { toCamelcase } from '../../utils/util'
+import { useInterval } from '../../utils/hook'
+import {
+  ConfirmationTimeFeeRateChart,
+  FeeRateCards,
+  FeeRateTransactionCountChart,
+  LastNDaysTransactionFeeRateChart,
+} from './FeeRateTrackerComp'
+import Loading from '../../components/Loading'
+
+const FeeRateTracker = () => {
+  const [fetchedTimeCounter, setFetchedTimeCounter] = useState<number>(0)
+
+  const { data: transactionFeesStatistic } = useQuery<FeeRateTracker.TransactionFeesStatistic>(
+    ['statistics-transaction_fees'],
+    () =>
+      v2AxiosIns.get(`statistics/transaction_fees`).then(({ status, data }) => {
+        if (status === 200 && data) {
+          setFetchedTimeCounter(0)
+          return toCamelcase<FeeRateTracker.TransactionFeesStatistic>(data)
+        }
+        return {
+          transactionFeeRates: [],
+          pendingTransactionFeeRates: [],
+          lastNDaysTransactionFeeRates: [],
+        }
+      }),
+    {
+      refetchInterval: 1000 * 60,
+    },
+  )
+
+  useInterval(() => {
+    setFetchedTimeCounter(c => c + 1)
+  }, 1000)
+
+  return (
+    <Content>
+      <div className={`${styles.feeRateTrackerPanel} container`}>
+        <Row className={styles.title}>Fee Rate Tracker</Row>
+        <Row className={styles.updatedTimeCounter}>Updated {fetchedTimeCounter}s ago</Row>
```

Comment:
It's better to use `Date.now() - lastUpdateTime` to get how long it's gone. Using `interval + update counter` is imprecise because `interval` hangs in some cases.

Reply:
And `requestAnimationFrame` is recommended to update data periodically

Ref: https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame